src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -11847,7 +11882,9 @@ def _create_tag_resource_schemas(
                                         )
                                     )
                     else:
-                        logger.debug(
+                        raise IllegalOperationError(
```

Comment:
`IllegalOperationError` is used for authorization errors. This feels more like a `ValueError`.

Reply:
Could you also use `ValueError` instead of  `IllegalOperationError` in `update_tag` please ?